shell/platform/android/test/io/flutter/view/AccessibilityBridgeTest.java

```diff
@@ -157,6 +158,16 @@ public void itHoverOverOutOfBoundsDoesNotCrash() {
     accessibilityBridge.onAccessibilityHoverEvent(MotionEvent.obtain(1, 1, 1, -10, -10, 0));
   }
 
+  @Test
+  public void releaseDropsChannelMessageHandler() {
+    AccessibilityChannel channel = mock(AccessibilityChannel.class);
+    AccessibilityBridge accessibilityBridge =
+        setUpBridge(null, channel, null, null, null, null);
+    verify(channel).setAccessibilityMessageHandler(null);
```

Comment:
since this is already null, I'd perhaps make this a bit more real by also injecting an AccessibilityManager and let it be isEnabled() true either via mockito or shadowOf(accessibilityManager).setEnabled(true)

Reply:
done